src/media/js/compatibility_filtering.js

```diff
@@ -36,27 +36,22 @@ define('compatibility_filtering',
     refresh_params();
     z.win.on('navigating', refresh_params);
 
-    // By default, filter by device everywhere but on desktop.
-    if (capabilities.firefoxOS) {
-        // Currently we don't distinguish between mobile & tablet on FirefoxOS,
-        // so don't set actual_device.
-        actual_dev = 'firefoxos';
-    } else if (capabilities.firefoxAndroid) {
-        actual_dev = 'android';
-        actual_device = capabilities.widescreen() ? 'tablet' : 'mobile';
-    }
+    // By default, filter by dev (platform) and device (form factor).
+    actual_dev = capabilities.device_platform();
```

Comment:
These two var names seem quite similar - this was existing code but could they be improved at the same time? If you're referring to just the var names later it's unclear what the difference between them is.